android/core/src/main/java/com/stadiamaps/ferrostar/core/NavigationViewModel.kt

```diff
@@ -60,22 +63,22 @@ interface NavigationViewModel {
 class DefaultNavigationViewModel(
     private val ferrostarCore: FerrostarCore,
     private val spokenInstructionObserver: SpokenInstructionObserver? = null,
-    locationProvider: LocationProvider
+    private val locationProvider: LocationProvider
 ) : ViewModel(), NavigationViewModel {
 
-  private var lastLocation: UserLocation? = locationProvider.lastLocation
+  private var snappedLocation: UserLocation? = locationProvider.lastLocation
```

Comment:
I don't think this is a valid default value. The location here is straight from the provider (unsnapped).